libs/string/src/lib.sw

```diff
@@ -25,8 +27,10 @@ impl String {
     /// # Arguments
     ///
     /// * `bytes` - The vector of `u8` bytes which will be converted into a `String`.
-    pub fn from_utf8(bytes: Vec<u8>) -> String {
-        String { bytes }
+    pub fn from_utf8(mut bytes: Vec<u8>) -> String {
```

Comment:
```suggestion
    pub fn from_utf8(mut bytes: Vec<u8>) -> Self {
```

Can you confirm or deny if this works? I think it should work and I have suggested it before.

Reply:
Whoops, missed these. Updated 👍